Dockerfile

```diff
@@ -42,6 +46,7 @@ RUN apt update && apt install -q -y freeglut3-dev
 RUN apt update && apt install -q -y python3 python3-distutils python3-pip
 RUN apt update && apt install -q -y libeigen3-dev
 RUN apt update && apt install -q -y libsqlite3-dev sqlite3
+RUN apt install -q -y libc6-dbg gdb valgrind
```

Comment:
gdb is now duplicated.

Probably line 116 should be removed.

Reply:
Fixed